install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -712,6 +712,10 @@ data:
   bgp-announce-lb-ip: {{ .Values.bgp.announce.loadbalancerIP | quote }}
 {{- end }}
 
+{{- if hasKey .Values "netfilterCompatMode" }}
+  netfilter-compatible-mode: {{ .Values.netfilterCompatMode | quote }}
+{{- end }}
```

Comment:
Without change in `values.yaml` then this value willl be undocumented. Is that intended?

Reply:
I was in impression documentation of a `flag` comes from this [file](https://github.com/cilium/cilium/blob/master/Documentation/cmdref/cilium-agent.md). Content in that file explains about the intention of flag. 
The reason i have added this flag in `cilium-configmap.yaml ` is to unit test my changes.

Reply:
@krishgobinath The values file defines the default value for the flag when installing via helm, and those are then automatically generated into  the helm options readme: https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/README.md .

However, to answer the original question I believe it's intentional for now to hide the flag while we sort out the details (& test and confirm the semantics we want).

Reply:
Thanks @joestringer for explanation.